analyses/doublet-detection/scripts/01b_run-scrublet.py

```diff
@@ -66,8 +66,8 @@ def main() -> None:
     args.results_dir.mkdir(parents = True, exist_ok = True)
 
     # Run scrublet and export the results
-    input_anndata = args.data_dir / (args.dataset_name + "_anndata.h5ad")
-    result_tsv = args.results_dir / (args.dataset_name + "_scrublet.tsv")
+    input_anndata = args.data_dir / (args.input_anndata_file)
```

Comment:
Same thing, I would just define this path outside of the script and take the full path as the input here.